examples/demo_threading/rust/src/workers/sensors.rs

```diff
@@ -69,7 +69,7 @@ impl SensorsWorker {
         network_rx: Receiver<NetworkChannel>,
         sensors: Arc<Mutex<Arc<SensorHashMap>>>,
         sensors_changed: Arc<AtomicBool>,
-        qt_thread: cxx::UniquePtr<EnergyUsageCxxQtThread>,
+        qt_thread: EnergyUsageCxxQtThread,
```

Comment:
```suggestion
        qt_thread: CxxQtThread<EnergyUsage>,
```

Reply:
It's not quite that though, it's `CxxQtThread<qobject::EnergyUsage>` or `CxxQtThread<EnergyUsageQt>` the latter of which is in scope already.

As we move to the new API the former will be easier to do due to the module layout.

Reply:
changed it to bring `qobject` in as the other file needed the change anyway.